app/templates/users.hbs

```diff
@@ -11,6 +11,6 @@
   showBulkNewUserForm=showBulkNewUserForm
   setShowBulkNewUserForm=(action (mut showBulkNewUserForm))
   searchTerms=searchTerms
-  setSearchTerms=(action (mut searchTerms))
+  setSearchTerms=(action (mut searchTerms) value="target.value")
```

Comment:
I think we can do without ` value="target.value"` here since it is specified on the input itself.